src/fundus/publishers/us/business_insider.py

```diff
@@ -15,20 +15,13 @@
 
 class BusinessInsiderParser(ParserProxy):
     class V1(BaseParser):
-        _summary_selector = CSSSelector("article div.bi-bulletpoints > p")
+        _summary_selector = CSSSelector("article ul.summary-list > li")
         _subheadline_selector = CSSSelector("article h2")
-
         _paragraph_selector = XPath(
             """
             //article 
-            //div[contains(@class, 'article-body')] 
-            //p[
-                not(
-                    ancestor::*[@class='bi-bulletpoints'] or
-                    mark[@class='has-inline-color has-cyan-bluish-gray-color'] or 
-                    @class='has-text-align-right'
-                )
-            ]
+            //div[contains(@class, 'content-lock-content')] 
+            /p[text() and not(contains(@class, 'disclaimer'))]
```

Comment:
The paragraph_selector does not seem to cover all cases: e.g. https://www.businessinsider.com/trump-civil-fraud-verdict-whats-in-judge-engorons-decision-2024-2 , https://www.businessinsider.com/couples-splurge-on-guest-experience-luxury-weddings-2024-2

Reply:
I removed `text()` from the selector.